openaerostruct/geometry/geometry_user_mesh_util.py

```diff
@@ -0,0 +1,661 @@
+""" Utility for quickly generating a multi-section user specificed OAS mesh
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def generateMesh(
+    sections, data, bPanels, cPanels, symmetry=True, controlPoints=False, generatePlots=False, plotOptions={}
+):
+    """
+    Quickly generates user multi-section mesh for analysis based on a few parameters
+
+    Parameters
+    ----------
+    sections : int
+        Integer for number of wing sections specified
+    data : numpy array
+        sectionsx4 array consisting of taper, root chord length, aspect ratio, and leading edge sweep in columns. Each row corresponds to each specified wing section.
+    bPanels: numpy array
+        1 x sections 1-D array consisting of integers that specify the number of spanwise panels corresponding to each wing section
+    cPanels: int
+        Integer for number of chord-wise panels
+    symmetry : boolean
+        Flag set to True if surface is reflected about y=0 plane.
+    controlPoints: boolean
+        Flag set to True if both quarter chord and three-quarter chord control points should be computed
+    generatePlots: boolean
```

Comment:
Instead of adding a mesh visualizer specific to this multi-section wing, what do you think of adding a plotter that works for any mesh: multi-section, CRM, rectangular wing, or an optimized one. Is there any specific visualization thing for multisection wings? The plotter function would take the OAS's 3D wing array (shape nx, ny, 3), symmetry flag, etc., as inputs.

I previously thought we wouldn't need to add a mesh plotter because that's very simple code, but now I think it'd be nice to add that in `openaerostruct/utils` and a quick document on that.